### DISCUSSION TITLE
可以使用connections.connect同时创建两个连接吗，让两个线程各使用一个连接

### DISCUSSION THREAD

**Discussion by zyy20191** [milvus-io/milvus#18089] [Q&A and General discussion] [answered]
我创建100个线程使用同一连接，发现查询速度非常慢，所以想问问能否创建多个连接，让线程使用不同的连接，提升查询速度。可以的话应该怎么设置呢

**Comment by yhmo** [comment, marked as answer]
可以不同的线程用不同的连接，参考这里：https://github.com/milvus-io/milvus/discussions/18034
